intellij-java-google-style.xml

```diff
@@ -267,6 +20,132 @@
   <option name="ALIGN_MULTILINE_EXTENDS_LIST" value="true" />
   <option name="ALIGN_MULTILINE_PARENTHESIZED_EXPRESSION" value="true" />
   <option name="ALIGN_MULTILINE_ARRAY_INITIALIZER_EXPRESSION" value="true" />
+  <option name="KEEP_BLANK_LINES_IN_CODE" value="1" />
+  <option name="BLANK_LINES_AFTER_CLASS_HEADER" value="1" />
+  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+  <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+  <option name="IMPORT_LAYOUT_TABLE">
```

Comment:
Any reason to move the IMPORT_LAYOUT_TABLE? It's hard to diff ... were there any changes?